src/components/institution/SelectInstitutionForm.tsx

```diff
@@ -150,35 +149,61 @@ export const SelectInstitutionForm = ({ onSubmit, onClose, suggestedInstitutions
               )}
             </Field>
             {values.unit?.hasPart && values.unit.hasPart.length > 0 && (
-              <Field name={SelectOrganizationFormField.Subunit}>
-                {({ field }: FieldProps<Organization>) => (
-                  <Autocomplete
-                    options={getSortedSubUnits(values.unit?.hasPart)}
-                    getOptionLabel={(option) => getLanguageString(option.labels)}
-                    renderOption={(props, option) => (
-                      <OrganizationRenderOption key={option.id} props={props} option={option} />
-                    )}
-                    onChange={(_, value) => setFieldValue(field.name, value)}
-                    filterOptions={(options, state) =>
-                      options.filter((option) =>
-                        state.getOptionLabel(option).toLocaleLowerCase().includes(state.inputValue.toLocaleLowerCase())
-                      )
-                    }
-                    renderInput={(params) => (
-                      <TextField
-                        {...params}
-                        data-testid={dataTestId.organization.subSearchField}
-                        label={t('registration.contributors.department')}
-                        variant="filled"
-                        fullWidth
+              <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem' }}>
+                <Typography variant="h3" component="h2" sx={{ marginTop: '1rem', fontWeight: 'normal' }}>
```

Comment:
Is `variant="h3" component="h2"` contradictory?

Reply:
Modalen har  `<h2>Legg til ny tilknytning</h2>`, så enig i at det gir mening om både "Velg institusjon" (i `AddAffiliationModal.tsx`) og "Velg enhet" her er h3, og dermed under headingen til selve modalen
```suggestion
                <Typography variant="h3" sx={{ marginTop: '1rem', fontWeight: 'normal' }}>
```

Reply:
Godt poeng. Jeg så dette gjort et annet sted og tenkte component=h2 betydde at semantisk skulle det være h2, men utseende-messig skulle det ha stylingen til h3. Men her er det ikke noen problem om det semantisk sett bør være h3 så endrer.